docs/source/driver/go/flight_sql.rst

```diff
@@ -80,63 +84,56 @@ API specification 1.0.0, as well as some additional, custom options.
 Authentication
 --------------
 
-The driver does no authentication by default.
+The driver does no authentication by default.  The driver implements a
+few optional authentication schemes:
 
-The driver implements one optional authentication scheme that mimics
-the Arrow Flight SQL JDBC driver.  This can be enabled by setting the
-option ``arrow.flight.sql.authorization_header`` on the
-:cpp:class:`AdbcDatabase`.  The client provides credentials by setting
-the option value to the value of the ``authorization`` header sent
-from client to server.  The server then responds with an
-``authorization`` header on the first request.  The value of this
-header will then be sent back as the ``authorization`` header on all
-future requests.
+- A user/password scheme: TODO
```

Comment:
this is implemented already, or are we waiting until the options are standardized in the docs for it?

Reply:
ah, I didn't get to actually test and document that

Reply:
I intend to keep iterating, so I just haven't added docs for it here

Reply:
(in future PRs)

Reply:
It's tested with the Dremio integration test

Reply:
Updated docs.